install-daemon.sh

```diff
@@ -155,6 +161,50 @@ function check_os_comp {
     print_error "Unsupported OS"
     exit 1
   fi
+  if [ "$OS" == "debian" ] || [ "$OS" == "ubuntu" ] || [ "$OS" == "zorin" ]; then
+    apt-get update
+
+    # install virt-what
+    apt-get install -y virt-what
+	
+   elif [ "$OS" == "centos" ]; then
+	
+    if [ "$OS_VER_MAJOR" == "7" ]; then
+      yum -y update
+
+      # install virt-what
+      yum -y install virt-what
+    elif [ "$OS_VER_MAJOR" == "8" ]; then
+      dnf -y update
+		
+      # install virt-what
+      dnf install -y virt-what
+    fi
+  else
+    print_error "Invalid OS."
+    exit 1
+  fi
+  echo -e "* ${COLOR_RED}Note${COLOR_NC} Detecting virtualization"
+  print_brake 70
+  
+  virt_serv=$(virt-what)
+  if [ "$virt_serv" = "" ]; then
+      print_warning "No virtualization detected"
+  else
+      print_warning "Virtualization: $virt_serv detected."
+  fi
+  
+  print_brake 70
+  
+  if [ "$virt_serv" == "openvz" ] || [ "$virt_serv" == "lxc" ] ; then # add more virtualization types which are not supported
+      print_warning "Unsupported type of virtualization detected. Please consult with your hosting provider whether your server can run Docker or not. Proceed at your own risk."
+      print_error "Installation aborted!"
+      exit 1
+  fi
+  if uname -r | grep -q "xxxx"; then
+      print_error "Unsupported kernel detected."
+      exit 1
+  fi
```

Comment:
This section is performed after the initial welcome message is printed meaning the whole output will just disappear since  `apt`/`dnf`/`yum` do print a lot of information when running. Can `check_os_comp` on line 446 be moved to after line 432?

Reply:
Yes, it can be moved without a problem. 👍